fetch/main.js

```diff
@@ -0,0 +1,21 @@
+async function fetchData() {
+  const response = await fetch('https://jsonplaceholder.typicode.com/users');
+  const users = await response.json();
+  console.log(users);
+}
+fetchData();
+
+async function getPokemon() {
+  try {
+    const response = await fetch('https://pokeapi.co/api/v2/pokemon/39');
+    if (!response.ok) {
+      throw new Error(`Error: , status code: ${response.status}`);
+    } else {
```

Comment:
Since you `throw` on line 12, you don't need to use an `else` clause. It doesn't hurt, it's just not necessary.